core/platform/users/gae_current_user_services_test.py

```diff
@@ -0,0 +1,86 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the Gae current user services."""
+
+from core.platform.users import gae_current_user_services
+from core.tests import test_utils
+
+from google.appengine.api import users
+
+
+class GaeCurrentUserServicesTest(test_utils.GenericTestBase):
+
+    def test_create_login_url(self):
+        login_url = gae_current_user_services.create_login_url('splash')
+        self.assertEquals(
```

Comment:
@apb7 it should be self.assertEqual here and elsewhere, I think. Can we add a lint check for this? I think self.assertEquals is deprecated.

@hemant0308 could we change to self.assertEqual throughout? See https://docs.python.org/2/library/unittest.html#deprecated-aliases

Reply:
@seanlip: Filed #5577 for the same. Thanks!

Reply:
changed in <b>core.users.gae_current_user_services_test.py</b>.
not touched other files as of now.

Reply:
Hi @hemant0308! You need not add a check for this. I have already done this in PR #5577. Just change all occurrences of `assertEquals` to `assertEqual` which you have added in this PR, in the test file only. Also, do not hesitate to ask away your queries. We are here to help :)

Reply:
I have changed in this PR files.
Thanks @apb7